IFC4x3/Properties/e/Emodulus_3hpNJVftrE1AuFMdtJ4v6C/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Emodulus_3hpNJVftrE1AuFMdtJ4v6C" Name="Emodulus" UniqueId="ebcd74df-a77d-4e04-ae0f-5a7dd313918c" PrimaryDataType="IfcPressureMeasure" />
```

Comment:
Use existing `YoungModulus`.

Reply:
> Use existing `YoungModulus`.

If it is tensile strength. Need to ask @j-weil if this is ok.

Reply:
Fixed in [25580c9](https://github.com/bSI-InfraRoom/IFC-Specification/pull/612/commits/25580c952e660a7aafce1122cb397281016824ed)

Reply:
This is resolved, right?